### PR TITLE
JS library - Top scenarios with Vanilla JS

### PR DESCRIPTION
**Pull request by promatik**
This PR is pointing to [`test-crud-field-js-library`](https://github.com/Laravel-Backpack/demo/pull/384) #384.

It aims the test of JS Fields with the vanilla JS code at CRUD https://github.com/Laravel-Backpack/CRUD/pull/4391.

### CODE REVIEW

File: public/assets/js/monster/fields.js [hunk 1]
```diff
@@ -35,7 +35,7 @@ var monsterFields = [
     'time', // time
     'upload', // upload
     'upload_multiple', // upload_multiple
-    'url', // url
```
Comment (promatik) [inline public/assets/js/monster/fields.js:38]: The URL field is removed in the MonsterCrudController.

File: public/assets/js/monster/fields.js [hunk 2]
```diff
@@ -45,7 +45,7 @@ var monsterFields = [
     'base64_image', // base64_image PRO
     'wysiwyg', // ckeditor PRO
     'color_picker', // color_picker PRO
-    'start_date', 'end_date', // date_range PRO
+    'start_date', //'end_date', // date_range PRO
```
Comment (promatik) [inline public/assets/js/monster/fields.js:48]: The field stands by 'start_date' only (either in the current vanilla JS solution, or the old one with jQuery).
We either change the query selector way to work, or we change the field.

File: public/assets/js/monster/test-top-scenarios.js [hunk 3]
```diff
@@ -1,136 +1,105 @@
- // EXAMPLE 1
- // MUST: when a checkbox is checked, show a second field;
- crud.field('visible').onChange(function(e, value) {
-    if (value == 1) {
-        crud.field('visible_where').show();
-    } else {
-        crud.field('visible_where').hide();
-    }
- });
+// EXAMPLE 1
```
Comment (promatik) [inline public/assets/js/monster/test-top-scenarios.js:1]: Sorry about the amount of changes, the spaces on the file were not correct 😕

**Comment by tabacitu** [comment]
Closing in favor of https://github.com/Laravel-Backpack/demo/pull/391

**Pull request closed without merging**